lib/github_api/client/projects/columns.rb

```diff
@@ -3,10 +3,15 @@
 
 module Github
   class Client::Projects::Columns < API
-    PREVIEW_MEDIA = "application/vnd.github.inertia-preview+json" # :nodoc:
     REQUIRED_COLUMN_PARAMS = %w(name).freeze
     REQUIRED_MOVE_COLUMN_PARAMS = %w(position).freeze
 
+    require_all 'github_api/client/projects/columns',
+                'cards'
+
+    # Access to Projects::Cards API
+    namespace :cards
```

Comment:
Sorry for not being specific enough. Your implementation is perfect and covers the whole api. What I meant by 'mimicking' was more in relation to how GitHub docs are structured. They have cards menu underneath the projects menu and that's why I thought it would be more 'natural' to have all the cards api requests under projects namespace:
```ruby
api.projects.cards.list 123 
api.projects.cards.list column_id: 123
api.projects.cards.get 123
api.projects.cards.get id: 123
etc.....
```
I think keeping the api navigation as shallow as we can would be good?